core/store/migrationsv2/1_initial.go

```diff
@@ -0,0 +1,3064 @@
+package migrationsv2
+
+import (
+	"github.com/jinzhu/gorm"
+	"gopkg.in/gormigrate.v1"
+)
+
+var (
+	up1 = `
```

Comment:
I wonder if we can load it from a `schema.sql` just to keep this clean?

Reply:
that's what I originally had - unfortunately in needs to be in the app code itself as we ship a single binary

Reply:
Would be nice to use this in future when go 1.16 is released:

https://tip.golang.org/pkg/embed/

Perhaps we can leave a card or a note?